src/components/Table/Table.types.part.tsx

```diff
@@ -185,6 +185,10 @@ export interface TableProps<T> extends StandardProps, ModeProviderProps<TableMod
    * Handler that is being called when a data cell has been clicked.
    */
   onDataClick?(e: TableCellEvent<T>): void;
+  /**
+   * Handler that is being called after the table has been sorted.
+   */
+  onSort?(e: TableCellEvent<T>): void;
```

Comment:
It would be great to have a dedicated type - `TableSortEvent`. I don't think it's a good idea to re-use the same event-type for completely different events.

Reply:
I though about that as well and then saw that every table event uses the same `TableCellEvent` interface so I figured it would be desired behavior to have the same event interface here as well. But I'm happy to create a custom event for it. :)

Reply:
Any other thoughts on this @guiherzog or @boromo?

Reply:
I've added now a custom `TableSortEvent`.